_articles/tutorial-download-data-portal.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Download Data Programmatically from Portals
+layout: article
+excerpt: Use R, Python or the command line to download data.
+category: in-practice
+---
+
+Portals allow you to explore data stored in [Synapse](https://sagebionetworks.org/tools_resources/synapse-platform/), a technology platform that allows researchers to aggregate, organize, analyze and share scientific data, code and insights.
+
+All entities in Synapse are automatically assigned a globally unique identifier used for reference with the format `syn12345678`. Often abbreviated to “synID”, the ID of an object never changes, even if the name does. 
+
+Synapse is designed to integrate seamlessly with your analytical workflow. Therefore, options to download data are available in the [R client]({{ site.baseurl }}{% link _articles/tutorial-download-data-portal.md %}#r), [command line client]({{ site.baseurl }}{% link _articles/tutorial-download-data-portal.md %}#command-line) and [Python client]({{ site.baseurl }}{% link _articles/tutorial-download-data-portal.md %}#python).
```

Comment:
I'm presuming this will render correctly once published, correct?

Reply:
correct.